packages/core/integration-tests/test/html.js

```diff
@@ -1554,7 +1554,7 @@ describe('html', function() {
     assert.equal(html.match(/<script/g).length, 4);
 
     html = await outputFS.readFile(path.join(distDir, 'e.html'), 'utf8');
-    assert.equal(html.match(/<script/g).length, 2);
+    assert.equal(html.match(/<script/g).length, 4);
```

Comment:
@wbinnssmith (this test has since changed) `e` now contains the `lodash/difference` shared bundle.